app/scripts/modules/core/loadBalancer/LoadBalancersTag.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from 'react';
+import * as ReactGA from 'react-ga';
+import autoBindMethods from 'class-autobind-decorator';
+import { sortBy } from 'lodash';
+
+import { Application } from '../application/application.model';
```

Comment:
Not a big deal, but it feels off that these two imports use relative paths and the ones below do not.